sstables/consumer.hh

```diff
@@ -114,8 +116,10 @@ private:
     } _read_int;
 
     // state for READING_BYTES prestate
+    size_t _read_bytes_len = 0;
     temporary_buffer<char> _read_bytes;
-    temporary_buffer<char>* _read_bytes_where; // which temporary_buffer to set, _key or _val?
+    std::vector<temporary_buffer<char>> _read_bytes_fragments;
```

Comment:
Suggest to use utils::small_vector<temporary_buffer<char>, 1> here, it can collapse _read_bytes and _read_bytes_fragments into one variable.

Reply:
std::vector is useful for the constructor of fragmented_temporary_buffer, where it can be std::moved, but we can collapse the variables anyway